content-src/components/DiscoveryStreamComponents/DSEmptyState/DSEmptyState.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+export class DSEmptyState extends React.PureComponent {
+  render() {
+    return (
+      <div className="section-empty-state">
+        <div className="empty-state-message">
+          <h2>You are caught up!</h2>
+          <div>Check back later for more stories.</div>
```

Comment:
Let's use `<p>` tag here…
```
<p>Check back later for more stories.</p>
```

Reply:
> Let's use `<p>` tag here…

sure, is paragraph tag also accessibility friendly?

Reply:
I believe so

Reply:
done